src/pytest_workflow/workflow.py

```diff
@@ -50,15 +51,19 @@ def __init__(self,
         # Always ensure a name. command.split()[0] can't fail because we tested
         # for emptiness.
         self.name = name or command.split()[0]
-        self.cwd = cwd
+        self.cwd = cwd or Path()
+        # For long running workflows it is best to save the stdout and stderr
+        # to a file which can be checked with ``tail -f``.
+        # stdout and stderr will be written to a tempfile if no CWD is given
+        # to prevent clutter created when testing.
+        self.stdout_file = (Path(tempfile.NamedTemporaryFile().name)
```

Comment:
In the case where `cwd` is `None`, I think it would be good idea to give the prefix and suffix an identifiable name. E.g.
```python
self.stdout_file = Path(tempfile.NamedTemporaryFile(prefix=self.name, suffix='.out').name)
```

Reply:
:+1: